test/e2e/tests/test_scenarios.py

```diff
@@ -128,3 +308,39 @@ class TestScenarios:
     """
     def test_scenario(self, scenario):
         runner.run(scenario)
+
+# assertTags returns AssertError if tags don't match
+def assertTags(expectedTags, latestTags):
+    for expectedTag in expectedTags:
+        tagKeyExist = False
+        for latestTag in latestTags:
+            if expectedTag == latestTag:
+                tagKeyExist = True
+                break
+        if tagKeyExist:
+            continue
+        assert expectedTag == None
```

Comment:
Maybe
```python
import collections
collections.Counter(x) == collections.Counter(y)
```

Reply:
expectTags is from manual input in yaml test file. In some cases, it might only list the part of modified tags instead of all tags (otherwise yaml test file may look too messy).